src/org/rascalmpl/debug/DebugHandler.java

```diff
@@ -234,25 +244,30 @@ public void processMessage(IDebugMessage message) {
 	      break;
 
 	    case RESUME:
-	      setSuspended(false);
-
-	      switch (message.getDetail()) {
-	      case STEP_INTO:
-	        setStepMode(DebugStepMode.STEP_INTO);
-	        getEventTrigger().fireResumeByStepIntoEvent();
-	        break;
-
-	      case STEP_OVER:
-	        setStepMode(DebugStepMode.STEP_OVER);
-	        getEventTrigger().fireResumeByStepOverEvent();
-	        break;
-
-	      case CLIENT_REQUEST:
-	        setStepMode(DebugStepMode.NO_STEP);
-	        getEventTrigger().fireResumeByClientRequestEvent();
-	        break;
-	      }
-	      break;
+			setSuspended(false);
+
+			switch (message.getDetail()) {
```

Comment:
the spacing got a bit messed up, makes it harder to review changes. can you undo that?